app/controllers/ep_postmaster/mailgun_hooks_controller.rb

```diff
@@ -30,7 +37,8 @@ def deliver_bounced_email_notification
           # REFACTOR: eventually remove these params in favor of original_message
           original_recipient: mailgun_post.recipient,
           original_subject: mailgun_post.subject,
-          error: mailgun_post.error }
+          error: EpPostmaster::SMTPError.normalize(mailgun_post.code, mailgun_post.error, mailgun_post.recipient)
```

Comment:
This looks ok here, but in the Relay PR, it looks like: `EpPostmaster::SMTPError.normalize(e.message, e.message, recipient)` which is a little funny to send the same param twice. Maybe we can refactor this `normalize` method to take two params?
```ruby
EpPostmaster::SMTPError.normalize(mailgun_post, mailgun_post.recipient)
```
And `normalize` could start out checking if `status_code.respond_to?(:code)` to see if we're working with a mailgun response or an error message (from the Relay).

Just a suggestion to avoid the `(e.message, e.message...` params.